docs/en/compute/Tensorflow.md

```diff
@@ -156,35 +156,35 @@ script [tf_cnn_benchmarsk.py](https://github.com/tensorflow/benchmarks/blob/mast
 
 ### Write Tensorflow applications with data location parameter
 
-When running Tensorflow on top of HDFS, S3, and other under storages, it requires to 
-configure Tensorflow and modify tensorflow applications. Through Alluxio POSIX API,
-users only need to mount their various under storages to Alluxio once and mount the 
-parent folder of those under storages that containing training data to the local filesystem.
-After mounting, data in various under storages become immediately available through Alluxio
-fuse mount point and can be transparently accessed to Tensorflow applications.
-If the Tensorflow application has the data location parameter, we only need to pass the 
+Running Tensorflow on top of HDFS, S3, and other under storages could require different 
+configurations, making it difficult to manage and integrate Tensorflow applications with 
+different under storages. Through Alluxio POSIX API,
+users only need to mount under storages to Alluxio once and mount the 
+parent folder of those under storages that contain training data to the local filesystem.
+After the initial mounting, the data becomes immediately available through Alluxio
```

Comment:
should there be an article before "fuse"? "an", "the", etc..?

Reply:
updated - added "the"